policyengine_us/variables/gov/hhs/medicaid/income/tax_unit_medicaid_income_level.py

```diff
@@ -10,10 +11,27 @@ class tax_unit_medicaid_income_level(Variable):
         "Medicaid/CHIP-related MAGI as fraction of federal poverty line."
         "Documentation: 'Federal poverty level (FPL)' at the following URL:"
         "URL: https://www.healthcare.gov/glossary/federal-poverty-level-fpl/"
+        "**Pregnant Women:**"
+        "  * Pregnant women are counted as themselves plus the number of children they are expecting to deliver"
+        "    when determining household size for Medicaid eligibility."
+        "  * Sources:"
+        "      URL: https://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11618&fileName=10%20CCR%25202505-10%208.100"
+        "      URL: https://www.cms.gov/marketplace/technical-assistance-resources/special-populations-pregnant-women.pdf"
     )
     definition_period = YEAR
 
     def formula(tax_unit, period, parameters):
         income = tax_unit("medicaid_magi", period)
-        fpg = tax_unit("tax_unit_fpg", period)
-        return income / fpg
+
+        members = tax_unit.members
+        pregnant_count = tax_unit.sum(
```

Comment:
don't need `members`
```suggestion
        pregnancies = add(tax_unit, period, ["pregnancies"])
```